src/exchanges/KyberAdapter.sol

```diff
@@ -185,7 +185,7 @@ contract KyberAdapter is DSMath, ExchangeAdapter {
     /// @param srcToken Address of src token
     /// @param destToken Address of dest token
     /// @param srcAmount Amount of src token
-    /// @return destAmount Amount of dest token expected in return
+    /// @param destAmount Amount of dest token expected in return
```

Comment:
This param is already defined in line 186. The `@return` should be for `minRate`